### PR TITLE
amule, toxic: depend on desktop-file-utils

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Since they install desktop files, utilize those.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: net/amule/Portfile [hunk 1]
```diff
@@ -62,3 +63,7 @@ configure.cxxflags-append \
                     "-framework CoreFoundation -framework CoreServices -framework IOKit"
 configure.ldflags-append \
                     "-framework CoreFoundation -framework CoreServices -framework IOKit"
+
+post-activate {
+    system "${prefix}/bin/update-desktop-database ${prefix}/share/applications"
+}
```
Comment (reneeotten) [inline net/amule/Portfile:69]: should this also be run when the port is uninstalled?

Comment (barracuda156) [reply net/amule/Portfile:69]: Hmm, honestly I do not know. I think other ports do not do that, but it may be wrong. What do you think? If yes, then how?

Comment (barracuda156) [reply net/amule/Portfile:69]: @reneeotten I am looking at existing ports now, and none seem to use `post-deactivate` closure for `desktop-file-utils`.

Comment (barracuda156) [reply net/amule/Portfile:69]: @reneeotten Maybe merge?

**Pull request merged**